flixel/addons/nape/FlxNapeSprite.hx

```diff
@@ -116,10 +121,14 @@ class FlxNapeSprite extends FlxSprite
 	{
 		super.revive();
 		
-		if (body != null)
+		if (body != null && body.compound == null)
```

Comment:
Are you sure it shouldn't look like this?

```haxe
if (body != null)
{
	if (body.compound == null)
		body.space = FlxNapeSpace.space;
	else
		body.compound.bodies.add(body);
}
````

Right now it would add `null` to `compound.bodies`.